File: beginner/1012.py
[a, b, c] = map(float, input().rsplit(" "))
pi = 3.14159

print(f"TRIANGULO: {(a * c) / 2:.3f}")
print(f"CIRCULO: {pi * (c ** 2):.3f}")
print(f"TRAPEZIO: {((a + b) / 2) * c:.3f}")
print(f"QUADRADO: {b ** 2:.3f}")
print(f"RETANGULO: {a * b:.3f}")

File: c/beginner/1012.c
#include <math.h>
#include <stdio.h>

#define pi 3.14159

int main(void) {
  double a, b, c;

  scanf("%lf %lf %lf", &a, &b, &c);
  printf("TRIANGULO: %.3lf\n", (a * c) / 2);
  printf("CIRCULO: %.3lf\n", pi * pow(c, 2));
  printf("TRAPEZIO: %.3lf\n", ((a + b) / 2) * c);
  printf("QUADRADO: %.3lf\n", pow(b, 2));
  printf("RETANGULO: %.3lf\n", a * b);

  return 0;
}
